### PR TITLE
Register + verify OTP email page

### PR DESCRIPTION
**Pull request by Tuan1O2**
- feat: add register + verify OTP email register page, add "Continue" btn in Welcome page, create Btn component, create regex + schemaYup.
 - refactor: re-custom back-button, add path register in Login
 - styles: fix logo register 480
 - refactor: delete back-button, update btn which pages are using, add .env in constants file
 - vendor: add react-hook-form-mui + yup libs.

### CODE REVIEW

**Review by MaiNhatHoangY2001 — APPROVED** [review]

File: src/components/ui/button/back-button.tsx
Comment (mailong395) [inline src/components/ui/button/back-button.tsx:1]: BackButton like to Btn. we need to merge them

File: src/components/ui/button/btn.tsx
Comment (mailong395) [inline src/components/ui/button/btn.tsx:1]: BackButton like to Btn. we need to merge them

File: src/pages/auth/verify-email/index.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ const VerifyEmail = () => {
   const {title, subTitle} = HEADER_VERIFY_EMAIL;
   return (
     <main className="max-w-md space-y-8 bg-black/5 p-8 rounded-3xl">
-      <BackButton />
+      <BackButton>Go Back</BackButton>
```
Comment (mailong395) [inline src/pages/auth/verify-email/index.tsx]: BackButton like to Btn. we need to merge them

File: src/utils/constants.ts
Comment (mailong395) [inline src/utils/constants.ts:1]: add .evn for this url

**Pull request merged**